src/redux/table/table.reducer.js

```diff
@@ -35,18 +34,28 @@ const tableState = (state = initialState, action = {}) => {
       ...state,
       pagination: { ...state.pagination, rowsPerPage: action.payload }
     };
-  case SET_PAGES_COUNT:
-    return {
-      ...state,
-      pagination: { ...state.pagination, pagesCount: action.payload }
-    };
   case SET_ITEMS_COUNT:
     return {
       ...state,
       itemsCount: action.payload
     };
   case RESET_PAGINATION:
     return initialState;
+  case UPDATE_PADAGINATION: {
```

Comment:
rename this case like 'UPDATE_PAGINATION'

Reply:
Fixed